UI/ActivityDialog.cs

```diff
@@ -15,7 +16,7 @@ public class ActivityDialog
         private const string RunActivityName = "Running";
         private const string ClimbActivityName = "Climbing";
 
-        private static string directoryPath = @"U:\Module C# 10 Fundamentals\FitnessApp\Data";
+        private static string directoryPath = @"D:\FitnessApp\Data";
```

Comment:
Make sure the app does not brake if this path does not exist. There is a way to create a directory if it does not exist.